src/OpenTelemetry.Exporter.Prometheus.HttpListener/CHANGELOG.md

```diff
@@ -8,6 +8,9 @@
   ([#3430](https://github.com/open-telemetry/opentelemetry-dotnet/pull/3430)
   [#3503](https://github.com/open-telemetry/opentelemetry-dotnet/pull/3503)
   [#3507](https://github.com/open-telemetry/opentelemetry-dotnet/pull/3507))
+* Fixed bug
```

Comment:
It will be great to have at least short description of the bug here.

Reply:
Done.